website/addons/wiki/routes.py

```diff
@@ -120,6 +120,23 @@
             '/project/<pid>/node/<nid>/wiki/<wname>/',
         ], 'delete', views.project_wiki_delete, json_renderer),
 
+        # Change Wiki Settings | PUT
+        Rule([
+            '/project/<pid>/wiki/settings/',
```

Comment:
The API V1 routes for addons seem somewhat inconsistent. Some use `/config/` and some use `/settings/`:

        Rule(
            ['/project/<pid>/dropbox/config/',
            '/project/<pid>/node/<nid>/dropbox/config/'],
            'put',
            views.config.dropbox_config_put,
            json_renderer
        ),
____________________________________
        Rule(
            [
                '/project/<pid>/dataverse/settings/',
                '/project/<pid>/node/<nid>/dataverse/settings/',
            ],
            'post',
            views.config.dataverse_set_config,
            json_renderer,
        )

That being said, the routes are mostly use `/config/` to change node settings and sometimes use `/settings/` for profile settings. So the *most* consistent would probably be changing this to `/config/`.